app/controllers/people_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PeopleController < ApplicationController
+  def index
+    @people = Person.all
+    @tasks = Task.all
+  end
+
+  def show
+    @person = Person.find(params[:id])
+  end
+
+  def tasks
+    @person = Person.find(params[:id])
+    tasks = Task.all
+    @tasks = tasks.where(person_id: @person.id)
```

Comment:
with the `has_many :tasks` in `Person` these two lines can just be `@tasks = @person.tasks`